.changeset/metal-apes-reply.md

```diff
@@ -0,0 +1,8 @@
+---
+'example-app': patch
```

Comment:
ah now i have to be annoying - the non-released packages should not have a changeset line :) so remove this one, and then I think it's good to go.